odk/src/main/resources/ODK.properties

```diff
@@ -0,0 +1,2 @@
+#custom properties for the ODK
+org.motechproject.odk.bundle.name=odk
```

Comment:
My search might be broken because of the very large page size for this PR, but I don't think we are using this anywhere? Moreover, is this supposed to be the actual name of the OSGi bundle? Is it even possible to have to configurable without rebuilding the bundle?